### ISSUE TITLE
In a generated pdf, an image is there which puts restriction on copying the characters

### ISSUE THREAD

**Issue by dilagurung** [eKoopmans/html2pdf#125]
In a generated pdf, an image is there which puts restriction on copying the characters. Is there any feature to do this or am i missing something.

**Comment by Sharvadze** [comment]
+1 
I'm also interested in this

**Comment by Vigneshsivaraj** [comment]
yeah thats the problem for me also,kindly give solution anyone :(

**Comment by eKoopmans** [comment]
Hi all, the way html2pdf works is by converting the HTML into an image, and then pasting that image into a PDF.

Currently there's no viable solution to avoid the image step, though I am investigating one possible option. For now it will have to stay as is.

**Comment by lveillard** [comment]
Hello there! 
First of all thanks for your amazing work 💪

Would be lovely to know if there are some news on this!  @eKoopmans 

Thanks!

**Comment by feliperaul** [comment]
Same here!
